src/constants.ts

```diff
@@ -108,19 +108,19 @@ export const LOCKING_WRAPPER_ADDRESS: { [chainId in ChainId]: string } = {
 }
 
 export const ROCKET_JOE_TOKEN_ADDRESS: { [chainId in ChainId]: string } = {
-  [ChainId.RINKEBY]: '0x24c34A486a8d90b70DfEC74B01d5646a2E21e035',
+  [ChainId.RINKEBY]: '0x3Be901d05584dc7900461a1b538ce24F7989cD65',
   [ChainId.FUJI]: '0x0000000000000000000000000000000000000000',
   [ChainId.AVALANCHE]: '0x5483ce08659fABF0277f9314868Cc4f78687BD08'
 }
 
 export const LAUNCH_EVENT_LENS_ADDRESS: { [chainId in ChainId]: string } = {
-  [ChainId.RINKEBY]: '0x039DA74A48E4e855946E1e55c7a55e61521b1B1C',
+  [ChainId.RINKEBY]: '0xfa31cDA014bA4e1A2CfF02EA94a7B9E6DB6BcbC5',
   [ChainId.FUJI]: '0x0000000000000000000000000000000000000000',
   [ChainId.AVALANCHE]: '0x039AA5d3FC07f6f265656793A8c437055b653d68'
 }
 
 export const ROCKET_JOE_STAKING_ADDRESS: { [chainId in ChainId]: string } = {
-  [ChainId.RINKEBY]: '0x65Cdf6A529f65b01F0605A11bee6F48Ae66293e9',
+  [ChainId.RINKEBY]: '0x74355E4732aEfeef2B190a531FD2194dBC5d9749',
```

Comment:
you want this to be `RocketJoeStaking_Proxy` address instead of `RocketJoeStaking_Implementation` so should be `0xc1108bda59F2A66843833d5B801bF40C966d34ca`

Reply:
ahh thank you!